crypto/nrf_oberon/include/ocrypto_constant_time.h

```diff
@@ -38,7 +38,7 @@ extern "C" {
 int ocrypto_constant_time_equal(const void *x, const void *y, size_t length);
 
 /**
- * Variable length compared to zero.
+ * Variable length compare to zero.
```

Comment:
comparison?

Reply:
This is an external file that we get from Oberon so if this is not a blocker I would propose to leave it as is.